src/Raven.Studio/typescript/common/queryUtil.ts

```diff
@@ -39,12 +39,22 @@ class queryUtil {
         const fromPart = `from index ${escapedIndexName}`;
         let wherePart = `where ${escapedFieldName} == ${escapedValueName}`;
         
-        if (indexName.startsWith("Auto") && (value.startsWith("{") || value === "NULL_VALUE")) {
+        if (indexName.startsWith("Auto") && 
+            ((value.startsWith("{") && queryUtil.hasUpperCase(value)) || value === "NULL_VALUE")) {
             wherePart = `where exact(${escapedFieldName} == ${escapedValueName})`;
         }
         
         return `${fromPart} ${wherePart}`;
     }
+
+    static hasUpperCase(value: string): boolean {
+        for (let i = 0; i < value.length; i++) {
+            if (/^[A-Z]$/.test(value[i])) {
```

Comment:
what about national characters like ŁÓDŹ?

Reply:
reverted to the 1'st commit

Reply:
so both comments are not addressed:
https://github.com/ravendb/ravendb/pull/14672#discussion_r936467702
https://github.com/ravendb/ravendb/pull/14672#discussion_r936531528